src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,302 @@
+package duke;
+
+import duke.task.*;
```

Comment:
Following java coding standards, `*` should be replaced with individual import statements